### PR TITLE
AB#9631 Added character names to the cast list on film and tv detail pages

### PR DESCRIPTION
**Pull request by Jason-1**
Co-authored-by: Louis <l0ud0gg@users.noreply.github.com>

ADO card: ☑️ [AB#9631](https://dev.azure.com/S72/SHIFT72/_workitems/edit/9631)

## Description of work
Current template does not display character names if they are input into admin. Film and TV detail pages have been updated to show character names alongside the cast member if populated.

### Affected Clients
 - All clients who use Kibble

## Checklist
- [ ] CI tests are passing Github actions (inc. linting)
- [ ] Key areas of the feature outlined for context and testing
- [ ] If there are designs for this work are they noted here and in the ADO card 
- [ ] Design review
- [ ] Have checked this at multiple screen resolutions and range of browsers
- [ ] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [ ] Updated changelog (if applicable)
- [ ] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

File: site/templates/film/item.jet [hunk 1]
```diff
@@ -71,7 +71,8 @@
                       <p>
                         {{range index, member := film.Cast}}
                           {{ path := "/search.html?q=" + member.Name }}
-                          <a href="{{ routeToPath(path) }}">{{member.Name}}</a>{{if index < len(film.Cast) - 1}}, {{end}}
+                          <a href="{{ routeToPath(path) }}">{{member.Name}}</a>{{if len(member.Character) > 0}} (as {{member.Character}}) {{end}}
```
Comment (PeculiarGoat) [inline site/templates/film/item.jet:74]: We can't assume that `as` is valid for every language.  You'll have to add a translation key for it or just remove it!

Comment (l0ud0gg) [reply site/templates/film/item.jet:74]: Good point

**Comment by PeculiarGoat** [comment]
@Jason-1 @l0ud0gg is this PR abandoned?

**Review by PeculiarGoat — APPROVED** [review]
cat1face

**Review by l0ud0gg — APPROVED** [review]

**Pull request merged**